src/httpd_jsonapi.c

```diff
@@ -1654,6 +1655,17 @@ queue_item_to_json(struct db_queue_item *queue_item, char shuffle)
   safe_json_add_string(item, "composer", queue_item->composer);
   safe_json_add_string(item, "genre", queue_item->genre);
 
+  /* these are represented as strings as json's spec for handling large ints is
```

Comment:
The id values are defined as strings in the forked-daapd JSON API (see README for e. g. the track object). The comment is therefor a bit misleading. We want it to be a string for consistency between the queue object and the track object and not because the JSON spec is not suited for large ints.

Reply:
I've made the `snprintf()` updates and pushed - regarding your comment here and to be:  the data value is just going across as was:
```
    "album_id": "4811135685787355000",
```
Is this was you meant or did you expect this to be:

```
    "album_id": "library:album:4811135685787355000",
```
?

Reply:
`"album_id": "4811135685787355000"` is correct.